test/controllers/api/deploys_controller_test.rb

```diff
@@ -1,10 +1,14 @@
 # frozen_string_literal: true
 require_relative '../../test_helper'
-
 SingleCov.covered!
 
 describe Api::DeploysController do
+  assert_route verb: "GET", path: "/api/deploys/active_count", to: 'api/deploys#active_count'
```

Comment:
remove verb + path keyword args for better readability ? ... ideally use `:get` instead of "GET"